frontend/src/components/layout/AppNavBar/AppNavBar.tsx

```diff
@@ -136,6 +136,11 @@ function DisposeProjectsDropdown() {
           View Projects
         </NavDropdown.Item>
       )}
+      {(keycloak.hasClaim(Claims.PROJECT_VIEW) || keycloak.hasClaim(Claims.ADMIN_PROJECTS)) && (
```

Comment:
I think  you could have just included it in the above section with a `<><NavDropdown.Item/><NavDropdown.Item/></>`